src/main/java/frc/robot/common/hardware/ColorSensorMuxed.java

```diff
@@ -44,6 +64,46 @@ public ColorSensorMuxed(int... ports) {
     lastColorRead = 0;
   }
 
+  public boolean configureMeasurementRates(MeasurementRate rate) {
+
+    ColorSensorMeasurementRate colorRate;
+    ProximitySensorMeasurementRate proxRate;
+    sensorPeriodInSeconds = rate.period;
+    boolean ret = true;
+
+    switch (rate) {
+      case kRate40Hz:
+        colorRate = ColorSensorMeasurementRate.kColorRate25ms;
+        proxRate = ProximitySensorMeasurementRate.kProxRate25ms;
+        break;
+      case kRate20Hz:
+        colorRate = ColorSensorMeasurementRate.kColorRate50ms;
+        proxRate = ProximitySensorMeasurementRate.kProxRate50ms;
+        break;
+      case kRate5Hz:
+        colorRate = ColorSensorMeasurementRate.kColorRate200ms;
+        proxRate = ProximitySensorMeasurementRate.kProxRate200ms;
+        break;
+      case kRate10Hz:
+      default:
+        colorRate = ColorSensorMeasurementRate.kColorRate100ms;
+        proxRate = ProximitySensorMeasurementRate.kProxRate100ms;
+        break;
+    }
+    for (int p : i2cPorts) {
+      if (setI2cPort(p)) {
+        sensors.configureProximitySensor(ProximitySensorResolution.kProxRes11bit, proxRate);
+        sensors.configureColorSensor(
+            ColorSensorResolution.kColorSensorRes20bit, colorRate, GainFactor.kGain3x);
```

Comment:
I think our color sensing got worse because the color measurement rate is dependent on the resolution according to the APDS-9151 datasheet. 20bit made the period 400ms which is slower than our default 100ms @ 18bit. I'll fix and merge into this PR.